spec/schema.json

```diff
@@ -16,53 +16,346 @@
     }
   ],
   "paths": {
-    "/request": {
+    "/status": {
+      "get": {
+        "summary": "Returns information describing the strike-off status of the company",
+        "produces": [
+          "application/json"
+        ],
+        "tags": [
+          "Strike Off Objections"
+        ],
+        "responses": {
+          "200": {
+            "description": "a Status object has been returned successfully",
+            "schema": {
+              "$ref": "#/definitions/StrikeOffStatus"
+            }
+          },
+          "404": {
+            "description": "not found"
+          }
+        }
+      }
+    },
+    "/": {
       "post": {
-        "summary": "Temporary end-point to prove sprint zero connectivity from the web app",
+        "summary": "Create a new strike-off objection request",
```

Comment:
I think 'request' should go here. In a few other places too. HTTP requests are coming in, as you pointed out, but our terminology now doesn't have 'objection requests'.

Reply:
updated